common/leica_sep.c

```diff
@@ -1,27 +1,34 @@
 // SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2021 Leica Geosystems AG
+ */
 
 #include <common.h>
 #include <watchdog.h>
 #include <console.h>
 #include <leica_sep.h>
 
-#define SEP_ACK_CMD   		"SE203:X\r\n"
-#define SEP_BOARD_ID  		"SC111:\r\n"
+#define SEP_ACK_CMD           "SE203:X\r\n\0"
+#define SEP_BOARD_ID          "SC111:\r\n\0"
+#define SEP_PRODUCTION_FLAG   "SC135:\r\n\0"
```

Comment:
why do you need trailing '\0'?

Reply:
Because I used them with `strcmp` in the previous implementation, now that we have `strncmp` the trailing is not necessary, but isn't it safer to keep it?

Reply:
With `""` it's already zero-terminated